parquet-column/src/main/java/org/apache/parquet/io/RecordConsumerLoggingWrapper.java

```diff
@@ -67,16 +74,20 @@ private String indent() {
     }
 
     private void log(Object value) {
-      logger.debug(indent() + value);
+      LOGGER.debug(indent() + value);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
     public void startGroup() {
-      if (DEBUG) ++indent;
-      if (DEBUG) log("<!-- start group -->");
+      if (DEBUG_ENABLED) {
+        ++indent;
+      }
+      if (DEBUG_ENABLED) {
```

Comment:
@lw-lin, it looks like there are still some if statements that can be combined. Please make sure you check through the code before the next round of review for these.